docs/serving/load-balancing/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Load balancing"
+weight: 30
+type: "docs"
+---
+
+You can configure load balancing on Knative, by placing the _Activator service_ in the request path to act as a load balancer.
+
+**NOTE:** To do this, you must first ensure that individual pod addressability is enabled.
+
+## Activator pod selection
+
+Activator services are scaled horizontally, so there may be multiple Activators in a deployment. In general, the system will perform best if the number of existing pods is larger than the number of Activators, and those numbers divide equally.
+<!--TODO(#2472): Add better documentation about what the activator is; explain the components of load balancing; maybe add a diagram-->
+
+Knative assigns a subset of Activators for each revision, depending on the revision size. More revision pods will mean a greater number of Activators for that revision.
+
+The Activator load balancing algorithm works as follows:
+
+- If concurrency is unlimited, the request is sent to the better of two random choices.
+- If concurrency is set to a value less or equal than 3, the activator will send the request to the first pod that has capacity. Otherwise, requests will be balanced in a round robin fashion.
```

Comment:
```suggestion
- If concurrency is set to a value less or equal than 3, the activator will send the request to the first pod that has capacity. Otherwise, requests will be balanced in a round robin fashion, with respect to container concurrency
```
Or some variation thereof. We will send only to the pods that have capacity, but we will pick them in a RR fashion.